src/envoy/mixer/http_filter.cc

```diff
@@ -96,7 +96,17 @@ class Config : public Logger::Loggable<Logger::Id::http> {
           __func__);
     }
 
-    http_control_ = std::make_shared<HttpControl>(mixer_server);
+    std::map<std::string, std::string> attributes;
+    if (config.hasObject("attributes")) {
+      for (const std::string& attr : config.getStringArray("attributes")) {
+        attributes[attr] = config.getString(attr);
+      }
+    } else {
+      log().warn("No attributes is specified in the config: {}", __func__);
```

Comment:
I'm also OK with restricting mixer attributes to be same for all service versions. The drawback is that we cannot do gradual config change for the mixer attributes. The whole idea of service versions is that we can customize what set of attributes to send for a subset of the service destination pods.